zh/deploy-failures.md

```diff
@@ -16,15 +16,27 @@ aliases: ['/docs-cn/tidb-in-kubernetes/dev/deploy-failures/']
 
 ```shell
 kubectl get tidbclusters -n ${namespace}
+kubectl describe tidbclusters -n ${namespace} ${cluster_name}
 kubectl get statefulsets -n ${namespace}
 kubectl describe statefulsets -n ${namespace} ${cluster_name}-pd
 ```
 
+创建备份恢复任务后，如果 Pod 没有创建，则可以通过以下方式进行诊断：
+
+{{< copyable "shell-regular" >}}
+
+```shell
+kubectl get backups -n ${namespace}
+kubectl describe backups -n ${namespace} ${backup_name}
+kubectl describe backupschedules  -n ${namespace} ${backupschedule_name}
+kubectl describe restores -n ${namespace} ${restore_name}
```

Comment:
get job, describe job

Reply:
addressed in https://github.com/pingcap/docs-tidb-operator/pull/639/commits/a1d59d7be5355268d71d96591f38a9178c7b9329